### PR TITLE
New tagline for birbs

### PR DESCRIPTION
**Pull request by AsafGerchak**
Added h2 to make site awesomer!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,6 +8,8 @@
 </head>
 <body>
   <h1>In the sky! It's a bird! It's a plane! No... I was right the first time, it's a bird and that's great I like birds</h1>
+  <h2>Oh HELLLLLLLLLL yeah birbs birbzzzz brabs</h2>
```
Comment (J-Seaton) [inline index.html]: This is good, but it doesn't mean anything.

File: index.html [hunk 2]
```diff
@@ -8,6 +8,8 @@
...
+
```
Comment (J-Seaton) [inline index.html:13]: An image of a bird could go here!

**Review by J-Seaton — CHANGES_REQUESTED** [review]
nice work, but need some changes

File: index.html (hunk 1 above)
Comment (AsafGerchak) [inline index.html]:
```suggestion
  <h2>Birds are loverly</h2>
```

**Review by J-Seaton — APPROVED** [review]
yay! good job!!

**Pull request merged**